src/SignalRServiceExtension/Config/ServiceHubContextStore.cs

```diff
@@ -50,14 +57,37 @@ private async Task<IServiceHubContext> GetFromLazyAsync(string hubName, (Lazy<Ta
             try
             {
                 var value = await pair.lazy.Value;
-                store.TryUpdate(hubName, (null, value), pair);
+                _weakTypedHubStore.TryUpdate(hubName, (null, value), pair);
                 return value;
             }
             catch (Exception)
             {
-                store.TryRemove(hubName, out _);
+                _weakTypedHubStore.TryRemove(hubName, out _);
                 throw;
             }
         }
+
+        private Task<ServiceHubContext<T>> GetAsync<THub, T>() where THub : ServerlessHub<T> where T : class
+        {
+            return _strongTypedHubServiceProvider.GetRequiredService<ServerlessHubContext<THub, T>>().HubContextTask;
+        }
```

Comment:
I prefer this signature for `GetAsync`. As it stands though it's never used, and private.